### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jellizaveta**
Added DNS filter structure in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,3 +18,89 @@ yarn run build
...
+     
+* [EasyPrivacy third-party international](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_thirdparty_international.txt)
+###  Cryptominers
+#### These filter lists are used against cryptominers
+
+* [AdGuard Base filter cryptominers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/cryptominers.txt)
+###  Exception
+#### This list is used to exclude domains that cause serious site breakages
+
+* [Exception rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exceptions.txt)
```
Comment (zloyden) [inline README.md:96]: This file is also needed to be added, It does the same goal.
```
https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exclusions.txt
```

**Review by zloyden — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -93,11 +93,12 @@ The output is written to `Filters/filter.txt`.
 ###  Exception
 #### This list is used to exclude domains that cause serious site breakages
 
+* [Exclusions rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exclusions.txt)
```
Comment (Alex-302) [inline README.md:96]: `Exclusions rules` => `Exclusion rules`, or `Exclusions`(and `Exceptions` below)

**Review by Alex-302 — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -18,3 +18,90 @@ yarn run build
 ```
 
 The output is written to `Filters/filter.txt`.
+
+###  DNS filter structure
```
Comment (vbagirov) [inline README.md:22]: Shouldn't it be level 2 header here? `##`

File: README.md [hunk 4]
```diff
@@ -18,3 +18,90 @@ yarn run build
...
+###  Ad servers
+#### These filter lists block domains that are used to provide and show advertisements on websites
+
+* [AdGuard Base filter](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/adservers.txt)
+
+* [AdGuard Base filter first-party servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/adservers_firstparty.txt)
+    
+* [AdGuard Base filter foreign servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/foreign.txt)
```
Comment (vbagirov) [inline README.md:30]: What exactly does `foreign` mean here?

File: README.md [hunk 5]
```diff
@@ -18,3 +18,90 @@ yarn run build
...
+    
+* [ABPindo third-party servers](https://raw.githubusercontent.com/ABPindo/indonesianadblockrules/master/src/advert/thirdparty.txt)
+    
+* [AdblockID](https://raw.githubusercontent.com/realodix/AdBlockID/master/src/adservers.adfl)
+    
+* [hostsVN](https://raw.githubusercontent.com/bigdargon/hostsVN/master/filters/adservers.txt)
+###  Tracking servers
+#### These filter lists block domains that are used for tracking and analytics
+    
+* [AdGuard Tracking Protection filter third-party trackers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/SpywareFilter/sections/tracking_servers.txt)
```
Comment (vbagirov) [inline README.md:76]: I suggest adding a long dash in such names:

AdGuard Tracking Protection filter — third-party trackers

Comment (vbagirov) [reply README.md:76]: same for all the other phrases, i.e. with "first-party servers"

File: README.md [hunk 6]
```diff
@@ -18,3 +18,90 @@ yarn run build
...
+    
+* [EasyPrivacy international trackers](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_trackingservers_international.txt)
+     
+* [EasyPrivacy third-party international](https://raw.githubusercontent.com/easylist/easylist/master/easyprivacy/easyprivacy_thirdparty_international.txt)
+###  Cryptominers
+#### These filter lists are used against cryptominers
+
+* [AdGuard Base filter cryptominers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/BaseFilter/sections/cryptominers.txt)
+###  Exception
+#### This list is used to exclude domains that cause serious site breakages
```
Comment (vbagirov) [inline README.md:94]: These lists are used to ...

Also, it's not clear to me what's the difference between them. Should we explain?

File: README.md [hunk 7]
```diff
@@ -18,3 +18,90 @@ yarn run build
...
+
+* [Exclusion rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exclusions.txt)
+* [Exception rules](https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/master/Filters/exceptions.txt)
+###  News Exchange
+#### This list is used to block teaser advertisements
```
Comment (vbagirov) [inline README.md:99]: It's probably better to change the header to "These lists are used to ..." anyway, despite there's only one list here yet

File: README.md [hunk 8]
```diff
@@ -18,3 +18,90 @@ yarn run build
...
+* [AdGuard Russian filter news exchange servers](https://raw.githubusercontent.com/AdguardTeam/AdguardFilters/master/RussianFilter/sections/news_exchange.txt)
+###  Regional lists
```
Comment (vbagirov) [inline README.md:101]: There are a bunch of language-specific filter lists above, shouldn't they be here?

**Review by vbagirov — CHANGES_REQUESTED** [review]

File: README.md (hunk 6 above)
Comment (jellizaveta) [reply README.md:94]: They provide the same function, I don't think there is any need to explain separately 
@zloyden @Alex-302 What do you think?

File: README.md (hunk 8 above)
Comment (jellizaveta) [reply README.md:101]: This filter list contains both advertising and tracking rules, perhaps it should be moved to the `Others` group. Where we have `Additional rules` with the same combined list of domains.

File: README.md (hunk 4 above)
Comment (jellizaveta) [reply README.md]: foreign includes domains of websites we have no language-specific filter for yet

**Review by Alex-302 — APPROVED** [review]

File: README.md (hunk 6 above)
Comment (zloyden) [reply README.md]: @vbagirov they do the same goal, but approach to the goal is different.

File: README.md (hunk 8 above)
Comment (zloyden) [reply README.md]: Yep, it looks ok.

**Review by vbagirov — APPROVED** [review]

**Pull request merged**